test/appium/tests/test_wallet.py

```diff
@@ -33,7 +39,11 @@ def test_wallet_error_messages(self):
         send_transaction.enter_recipient_address_button.click()
         send_transaction.enter_recipient_address_input.set_value(incorrect_public_key)
         send_transaction.done_button.click()
-        send_transaction.error_dialog.wait_for_error_message('Invalid address')
+        error_message = 'Invalid address'
+        try:
+            send_transaction.error_dialog.wait_for_error_message(error_message)
+        except TimeoutException:
+            self.errors.append(error_message + ' error did not appear')
```

Comment:
@yevh-berdnyk I put it here because I really believe we should not pass `errors` into the page objects. It should be done on the test level